Gruntfile.js

```diff
@@ -289,17 +289,16 @@ module.exports = function (grunt) {
     //   dist: {}
     // },
 
-    imagemin: {
-      dist: {
-        files: [{
-          expand: true,
-          cwd: '<%= yeoman.app %>/images',
-          src: '{,*/}*.{png,jpg,jpeg,gif}',
-          dest: '<%= yeoman.dist %>/images'
-        }]
-      }
-    },
-
+//    imagemin: {
```

Comment:
Let's either remove this all together, or comment _why_ its commented out.  This was blowing up due to some lib not being available in Alpine?